index.js

```diff
@@ -76,7 +76,7 @@ module.exports = function(gitPath) {
     if (fs.existsSync(headFilePath)) {
       var branchSHA;
       var headFile = fs.readFileSync(headFilePath, {encoding: 'utf8'});
-      var branchName = headFile.split('/').slice(-1)[0].trim();
+      var branchName = headFile.split('/').slice(2).join("/").trim();
```

Comment:
Because as far as I can tell, that wouldn't work. I even tested it in the Node console, as follows:  

```
> var headString = "refs/heads/branch/with/slashes"
> var branchName = headString.split('/', 3)[2].trim()
> console.log(branchName)  // 'branch'
```